VirtualMachineService/VirtualMachineHandler.py

```diff
@@ -662,6 +662,12 @@ def create_and_deploy_playbook(self, public_key, playbooks_information, openstac
             osi_key_dict[openstack_id]["status"] = self.ACTIVE
         return status
 
+    def exist_server(self, name):
+        if self.conn.compute.find_server(name) is not None:
```

Comment:
I would change it to compute.get_server(SERVER_ID)

Find server would return true if any server would have the name.
get_server by id is unique